Slim/Plugin/WiMP/ProtocolHandler.pm

```diff
@@ -213,7 +213,7 @@ sub _getTrack {
 				#if ( main::DEBUGLOG && $log->is_debug ) {
 				#	$log->debug( 'getTrack ok: ' . Data::Dump::dump($info) );
 				#}
-
+				$info->{url} =~ s/^http:/https:/;
```

Comment:
This certainly does not belong in a PR 😉

Reply:
indeed

Reply:
As you said, the header is not processed anymore when we use S:P:P:HTTPS, so we bitbrate is missing and seeking pretty wrong. I'll see if there is an easy solution